README.md

```diff
@@ -1788,6 +1789,43 @@ Map::from( ['a' => 'X', 'b' => 'Y'] )->has( 'X' );
 ```
 
 
+### if()
+
+Executes callables $then and $else depending on callable $condition.
+
+```php
+public function if( $condition, Callable $then, $Callable $else = null ) : self
```

Comment:
`callable` should be lower case (it's an internal PHP type like "string") and the `$` removed before `$Callable`

Reply:
Resolved in 4e5cf55e0d52299bd0c86f499090473f0c5ab578